nasa/harmony-regression-tests
# Fix failures

## Description

This PR does 2 things:

* Disables tests for the Variable Subsetter service in UAT and SIT, as there are no longer any collections associated with this service in those environments. (If this continues to be the case, it is likely a good argument to remove that service chain)
* Updates a bunch of dependencies, including Python versions, where possible. Some environments have not been updated past 3.9, due to dependency conflicts. I didn't adopt 3.11 as this caused issues with `notebook`.

## Jira Issue ID

N/A

## Local Test Steps

* Pull this branch locally.
* Rebuild all the Docker image (`make images`).
* Set the necessary environment variables (testing against UAT).
* Run all the notebooks (`./run_notebooks.sh`).
* The notebooks should all pass.

## PR Acceptance Checklist
* ~~Acceptance criteria met~~
* ~~Documentation updated (if needed)~~
